src/main/java/com/ibanity/signature_service/SignatureServiceApplication.java

```diff
@@ -0,0 +1,13 @@
+package com.ibanity.signature_service;
```

Comment:
I was never personally fan of underscore in package names.
would you consider like
* com.ibanity.services.signature.http
could derive to 
* com.ibanity.services.signature.jws